rotkehlchen/api/rest.py

```diff
@@ -2362,17 +2362,18 @@ def get_aave_balances(self) -> dict[str, Any]:
         )
 
     @async_api_call()
-    def get_aave_stats(
+    def get_module_stats_using_balances(
```

Comment:
what do you mean by the using_balances part?

Reply:
because those two also use the defi balances to provide the statistics. The liquity module for example only uses the events